widgetssdk/src/main/java/com/glia/widgets/call/CallIntentBuilder.kt

```diff
@@ -0,0 +1,26 @@
+package com.glia.widgets.call
+
+import android.content.Context
+import android.content.Intent
+import com.glia.widgets.GliaWidgets
+
+internal data class CallIntentBuilder(private val context: Context) {
```

Comment:
Since this class is only for internal usage and used only to build intent, I'd suggest converting it to a higher-level function like
```kotlin
fun buildCallIntent(context: Context, configuration: Configuration): Intent {
// build intent here
} 
```

Reply:
Something like this?
https://github.com/salemove/android-sdk-widgets/commit/d76bb9ca4f8754e5730ce6796e5e33de308c6d18